app/options/HTTPClient.cue

```diff
@@ -17,6 +17,10 @@ HTTPClient: schema.#optionsGroup & {
 	]
 
 	options: {
+		domain: {
```

Comment:
I have a feeling you mixed up 2 things here

http-client is something else -- an actual HTTP Client we use to execute outbound requests